src/index.ts

```diff
@@ -1,3 +1,3 @@
-export { MagicAdminSDK as Magic } from './core/sdk';
+export { MagicAdminSDK } from './core/sdk';
```

Comment:
Please revert this change! This would cause a breaking change to the public API!